packages/davinci-client/src/lib/davinci.utils.ts

```diff
@@ -41,7 +41,9 @@ export function transformSubmitRequest(node: ContinueNode): DaVinciRequest {
   const formData = collectors?.reduce<{
     [key: string]: string | number | boolean | (string | number | boolean)[];
   }>((acc, collector) => {
-    acc[collector.input.key] = collector.input.value;
+    if (collector.input.key !== undefined) {
```

Comment:
I'm wondering why we need this. If we are filtering by category, and no collector within these categories should have an `undefined`, did we get something wrong?